bitnami/zookeeper/Chart.yaml

```diff
@@ -21,4 +21,4 @@ name: zookeeper
 sources:
   - https://github.com/bitnami/bitnami-docker-zookeeper
   - https://zookeeper.apache.org/
-version: 7.1.2
+version: 7.2.0
```

Comment:
Could you fix the lint error? It is probably happening due to the text editor, if you simply save the file with another one (i.e. `vim` or `emacs`) it should fix the issue.